PolicyServicePkg/PolicyService/DxeMm/PolicyCommon.c

```diff
@@ -311,15 +323,216 @@ CommonSetPolicy (
     Entry->AllocationSize = PolicySize;
     CopyMem (Entry->Policy, Policy, PolicySize);
     InsertTailList (&mPolicyListHead, &Entry->Link);
+    Status = EFI_SUCCESS;
   }
 
-  Status = InstallPolicyIndicatorProtocol (PolicyGuid);
-  if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_ERROR, "%a: Failed to install notification protocol. (%r)\n", __FUNCTION__, Status));
-    goto Exit;
+  Events = POLICY_NOTIFY_SET | (Entry->Attributes & POLICY_ATTRIBUTE_FINALIZED ? POLICY_NOTIFY_FINALIZED : 0);

```

Comment:
There is a path here where Entry will be NULL/Uninitialized. (if GetPolicyEntry returns an error)

Reply:
In the error case Entry will be allocated in the Else statement above, right?